lms/djangoapps/bulk_email/tests/test_email.py

```diff
@@ -201,6 +203,56 @@ def test_send_to_staff(self):
             [self.instructor.email] + [s.email for s in self.staff]
         )
 
+    def test_send_to_cohort(self):
+        """
+        Make sure email sent to a cohort goes there.
+        """
+        cohort = CourseCohort.create(cohort_name='test cohort', course_id=self.course.id)
+        for student in self.students:
+            add_user_to_cohort(cohort.course_user_group, student.username)
+        # Now we know we have pulled up the instructor dash's email view
+        # (in the setUp method), we can test sending an email.
+        test_email = {
+            'action': 'Send email',
+            'send_to': '["cohort:{}"]'.format(cohort.course_user_group.name),
+            'subject': 'test subject for cohort',
+            'message': 'test message for cohort'
+        }
+        # Post the email to the instructor dashboard API
+        response = self.client.post(self.send_mail_url, test_email)
+        # from nose.tools import set_trace; set_trace()
```

Comment:
The difference (in my case, at least) is muscle memory - this is what I learned first when picking up python. I really ought to switch though, saves a few keystrokes.